dkobo/koboform/api.py

```diff
@@ -52,6 +83,30 @@ def delete_survey_draft(self, request, pk=None):
         draft = self.get_object()
         draft.delete()
 
+    def list(self, request, *args, **kwargs):
+        email = request.QUERY_PARAMS.get('email')
+        token = request.QUERY_PARAMS.get('token')
+        username = request.QUERY_PARAMS.get('username')
+        if email and token and username:
+            try:
+                user = User.objects.get(username=username)
+            except User.DoesNotExist:
+                user = User(username=username, password=username)
+                user.email = email
```

Comment:
can you set the email in the initializer above? Can you also call `create` above, or another helper, instead of `save` below?